osaka/__init__.py

```diff
@@ -3,6 +3,6 @@
 from __future__ import division
 from __future__ import absolute_import
 
-__version__ = "1.3.1"
+__version__ = "1.3.2"
```

Comment:
This can remain at 1.3.1 since we have not yet cut a new release of HySDS Core and we had already bumped this in the previous PR post v6.0.1 release.